lahja/endpoint.py

```diff
@@ -139,6 +183,15 @@ def logger(self) -> logging.Logger:
     def name(self) -> str:
         return self._name
 
+    @property
+    def subscribed_events(self) -> Set[Type[BaseEvent]]:
+        "returns the set of events this Endpoint is currently listening for"
```

Comment:
*Nitpick*: [We use imperative, present tense](https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/documentation.md#grammar) (same as Python docs)

>Return the set of events this Endpoint is currently listening for